doc/source/projections.rst

```diff
@@ -10,23 +10,41 @@ Managing Projections
 =========================================
 
 
-
 Coordinate Reference Systems
 -----------------------------
 
 CRS are important because the geometric shapes in a GeoSeries or GeoDataFrame object are simply a collection of coordinates in an arbitrary space. A CRS tells Python how those coordinates related to places on the Earth.
 
-CRS are referred to using codes called `proj4 strings <https://en.wikipedia.org/wiki/PROJ.4>`_. You can find the codes for most commonly used projections from `www.spatialreference.org <http://spatialreference.org/>`_.
+You can find the codes for most commonly used projections from `www.spatialreference.org <http://spatialreference.org/>`_.
+
+The same CRS can often be referred to in many ways. For example, one of the most commonly used CRS is the WGS84 latitude-longitude projection. This can be referred to using the authority code ``"EPSG:4326"``.
+
+*geopandas* can accept anything accepted by `pyproj.CRS.from_user_input() <https://pyproj4.github.io/pyproj/stable/api/crs.html#pyproj.crs.CRS.from_user_input>`_:
+
+- CRS WKT string
+- An authority string [i.e. "epsg:4326"]
+- An EPSG integer code [i.e. 4326]
+- A ``pyproj.CRS``
+- An object with a to_wkt method.
+- PROJ string
+- Dictionary of PROJ parameters
+- PROJ keyword arguments for parameters
+- JSON string with PROJ parameters
+
+For reference, a few very common projections and their EPSG codes:
+
+* WGS84 Latitude/Longitude: ``"EPSG:4326"``
+* UTM Zones (North): ``"EPSG:32633"``
+* UTM Zones (South): ``"EPSG:32733"``
+
 
-The same CRS can often be referred to in many ways. For example, one of the most commonly used CRS is the WGS84 latitude-longitude projection. One `proj4` representation of this projection is: ``"+proj=longlat +ellps=WGS84 +datum=WGS84 +no_defs"``. But common projections can also be referred to by `EPSG` codes, so this same projection can also called using the `proj4` string ``"+init=epsg:4326"``.
+What is the best format to store the CRS information?
+-----------------------------------------------------
 
-*geopandas* can accept lots of representations of CRS, including the `proj4` string itself (``"+proj=longlat +ellps=WGS84 +datum=WGS84 +no_defs"``) or parameters broken out in a dictionary: ``{'proj': 'latlong', 'ellps': 'WGS84', 'datum': 'WGS84', 'no_defs': True}``). In addition, some functions will take `EPSG` codes directly.
+PROJ strings can be lossy for storing CRS information. If you can avoid it, it is best to not use them. Additionally, PROJ strings will likely not be supported in future major version of PROJ for storing CRS information.
 
-For reference, a few very common projections and their proj4 strings:
+More info: https://proj.org/faq.html#what-is-the-best-format-for-describing-coordinate-reference-systems
```

Comment:
Can we mention one recommendation what to use here? Rather than saying "do not use PROJ strings, for more go somewhere else"?

Reply:
That is a good idea. I updated it.